src/test/resources/application-test.yml

```diff
@@ -50,7 +50,7 @@ spring:
           jwk-set-uri: ${api.base.url.oauth}/.well-known/jwks.json
 
   datasource:
-    url: jdbc:aws-wrapper:postgresql://localhost:5433/datashare?sslmode=disable
+    url: jdbc:aws-wrapper:postgresql://localhost:5433/datashare?sslmode=disable&wrapperPlugins=%22%22
```

Comment:
Trialling this, based on https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/332/files#diff-7fb6e193c0242a9ce1d71354abdb588575a6cc2391ea0c1cff109790686977b7R163-R169